src/main/java/com/mineinabyss/geary/ecs/actions/SwitchToTargetAction.kt

```diff
@@ -0,0 +1,19 @@
+package com.mineinabyss.geary.ecs.actions
+
+import com.mineinabyss.geary.ecs.GearyEntity
+import com.mineinabyss.geary.ecs.components.Target
+import com.mineinabyss.geary.ecs.components.get
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@SerialName("on_target")
```

Comment:
I'd prefer `on.target` as the name. I think underscore should be a replacement for space while a dot is for grouping similar thing (kinda like a package I guess). Since we'll have actions to switch to many different things, I think the dot makes more sense.